src/rgw/driver/rados/rgw_rest_pubsub.cc

```diff
@@ -652,7 +653,7 @@ void RGWPSCreateNotifOp::execute(optional_yield y) {
   if(configurations.list.empty()) {
     // get all topics on a bucket
     rgw_pubsub_bucket_topics bucket_topics;
-    op_ret = b.get_topics(&bucket_topics);
+    op_ret = b.get_topics(dpp, &bucket_topics);
```

Comment:
we don't need to add `dpp` as a member variable. `RGWPSCreateNotifOp` inherits from `RGWOp` which itself implements `DoutPrefixProvider`, so you can replace `dpp` with `this` in these calls to `b.get_topics()`